roles/zuul/templates/etc/zuul/zuul.conf

```diff
@@ -19,24 +19,37 @@ start = {{ zuul_gearman_server_start }}
 
 {% endfor %}
 
+{% if not zuul_zuul_v3 %}
 {% for name, details in zuul_sites.items() %}
 [site "{{ name }}"]
 {% for key, value in details.items() %}
 {{ key }} = {{ value }}
 {% endfor %}
 
 {% endfor %}
+{% endif %}
 
 [zuul]
+{% if not zuul_zuul_v3 %}
 layout_config = /etc/zuul/config/layout.yaml
+{% endif %}
 log_config = /etc/zuul/server-logging.conf
```

Comment:
the logging config ends up being installed as either server-* or scheduler-* depending on version, so this needs to updated to set it accordingly.